MAKE/Makefile.sisu_gcc

```diff
@@ -85,6 +85,6 @@ INC_EIGEN = -I$(LIBRARY_PREFIX)/eigen/
 INC_DCCRG = -I$(LIBRARY_PREFIX)/dccrg_new_neighbours/
 INC_VECTORCLASS = -I$(LIBRARY_PREFIX)/vectorclass
 #INC_FSGRID = -I$(LIBRARY_PREFIX)/fsgrid
-INC_FSGRID = -I/homeappl/home/koskelat/lib/fsgrid/
```

Comment:
Should this be the main one, and not the personak copies

Reply:
@tkoskela can oyu comment on the two simultaneous different INC_DCCRG here?